packages/flame_forge2d/lib/forge2d_camera.dart

```diff
@@ -22,4 +42,46 @@ class Forge2DCamera extends Camera {
   Vector2 scaleVector(Vector2 worldCoordinates) {
     return worldCoordinates * zoom;
   }
+
+  @override
+  void update(double dt) {
+    if (_targetBody != null) {
+      if (follow == null) {
+        follow = _targetBody!.position.clone()..y *= -1;
+      } else {
+        follow!
+          ..setFrom(_targetBody!.position)
+          ..y *= -1;
+      }
+    }
+    super.update(dt);
+  }
```

Comment:
Shouldn't this take into account the world bounds too?

Reply:
I think the normal camera already takes care of that?